tests/base_testcase.py

```diff
@@ -25,7 +25,10 @@ def setUp(self):
         #define an existing user
         self.saved_user = User("tyrion", "tyrion@gmail.com",
```

Comment:
You can explore the use of [Faker](https://github.com/joke2k/faker) to generate fake data instead of hardcoding.

```python
from faker import Faker
fake = Faker()
.
.
.
self.saved_user = User(fake.name(), fake.email())
```